hivemq-edge/src/frontend/.prettierignore

```diff
@@ -7,3 +7,4 @@ coverage/
 public/
 src/api/__generated__
 .gradle/
+package.json
```

Comment:
Don't exclude `package.json` from prettier. Instead, make sure `prettier --write .` is run at every file save (or manually once before committing)